dev/build.image/build.gradle

```diff
@@ -636,6 +637,21 @@ if (isAutomatedBuild && !isIFIXBuild) {
             }
         }
     }
+    
+    //currently noship
+    task packageOpenLibertyMicroProfile7(type: PackageLibertyWithFeatures) {
+        dependsOn parent.subprojects.assemble
+        dependsOn ':com.ibm.websphere.appserver.features:publishFeatureResources'
+        withFeatures this.&microProfile7Features
+        packageServerConflict = "io.openliberty.microProfile,io.openliberty.mpConfig,io.openliberty.mpTelemetry,io.openliberty.org.eclipse.microprofile.config,io.openliberty.mpCompatible,io.openliberty.internal.mpVersion"
```

Comment:
We think there shouldn't be this many features here. MP7, and therefore the MP7 zip, shouldn't have multiple versions of MP features in it.

Reply:
will address in a follow up